dydx-testnet-4/pregenesis.json

```diff
@@ -0,0 +1,1750 @@
+                  "maker_volume_share_requirement_ppm": 0,
+                  "maker_fee_ppm": 100,
+                  "taker_fee_ppm": 500
+                },
+                {
+                  "name": "2",
+                  "absolute_volume_requirement": "1000000000000",
+                  "total_volume_share_requirement_ppm": 0,
+                  "maker_volume_share_requirement_ppm": 0,
+                  "maker_fee_ppm": 100,
+                  "taker_fee_ppm": 450
+                },
+                {
+                  "name": "3",
+                  "absolute_volume_requirement": "5000000000000",
+                  "total_volume_share_requirement_ppm": 0,
+                  "maker_volume_share_requirement_ppm": 0,
+                  "maker_fee_ppm": 50,
+                  "taker_fee_ppm": 400
+                },
+                {
+                  "name": "4",
+                  "absolute_volume_requirement": "25000000000000",
+                  "total_volume_share_requirement_ppm": 0,
+                  "maker_volume_share_requirement_ppm": 0,
+                  "maker_fee_ppm": 0,
+                  "taker_fee_ppm": 350
+                },
+                {
+                  "name": "5",
+                  "absolute_volume_requirement": "125000000000000",
+                  "total_volume_share_requirement_ppm": 0,
+                  "maker_volume_share_requirement_ppm": 0,
+                  "maker_fee_ppm": 0,
+                  "taker_fee_ppm": 300
+                },
+                {
+                  "name": "6",
+                  "absolute_volume_requirement": "125000000000000",
+                  "total_volume_share_requirement_ppm": 5000,
+                  "maker_volume_share_requirement_ppm": 0,
+                  "maker_fee_ppm": -50,
+                  "taker_fee_ppm": 250
+                },
+                {
+                  "name": "7",
+                  "absolute_volume_requirement": "125000000000000",
+                  "total_volume_share_requirement_ppm": 5000,
+                  "maker_volume_share_requirement_ppm": 10000,
+                  "maker_fee_ppm": -70,
+                  "taker_fee_ppm": 250
+                },
+                {
+                  "name": "8",
+                  "absolute_volume_requirement": "125000000000000",
+                  "total_volume_share_requirement_ppm": 5000,
+                  "maker_volume_share_requirement_ppm": 20000,
+                  "maker_fee_ppm": -90,
+                  "taker_fee_ppm": 250
+                },
+                {
+                  "name": "9",
+                  "absolute_volume_requirement": "125000000000000",
+                  "total_volume_share_requirement_ppm": 5000,
+                  "maker_volume_share_requirement_ppm": 40000,
+                  "maker_fee_ppm": -110,
+                  "taker_fee_ppm": 250
+                }
+              ]
+            }
+          },
+          "block_height": 6912000
```

Comment:
@teddyding do we want this delay msg to kick in shortly after the testnet start time?